libs/libGamedata/lua/LuaInterfaceBase.cpp

```diff
@@ -53,37 +53,32 @@ void LuaInterfaceBase::Register(kaguya::State& state)
 
 void LuaInterfaceBase::ErrorHandlerNoThrow(int /*status*/, const char* message)
 {
-    LOG.write(_("Lua error: %s\n")) % (message ? message : "Unknown");
+    LOG.write(_("Lua error: %s\n")) % (message ? message : _("Unknown"));
```

Comment:
should'nt be this also "Unknown error"? to simplify translation?

Reply:
Yes, done